pyproject.toml

```diff
@@ -101,15 +101,6 @@ markers = [
     "gpu: tests that use a GPU (currently unused, but needs to be specified here as we import anndata.tests.helpers, which uses it)",
 ]
 
-[tool.hatch.envs.hatch-test]
-features = ["test", "rapids12"]
-
-[tool.hatch.envs.docs]
-installer = "uv"
-features = ["doc"]
-scripts.build = "sphinx-build -M html docs docs/_build {args}"
-scripts.clean = "git clean -fXd docs"
-
```

Comment:
why?

Reply:
I created a hatch.toml like scanpy and anndata has.

Reply:
oof, seems like I’m more jetlagged than I thought. OK, sure haha